tests/foreman/api/test_registration.py

```diff
@@ -227,3 +227,26 @@ def test_negative_global_registration_without_ak(module_target_sat):
     with pytest.raises(HTTPError) as context:
         module_target_sat.api.RegistrationCommand().create()
     assert 'Missing activation key!' in context.value.response.text
+
+
+def test_positive_capsule_without_registration_module(module_target_sat, module_capsule_configured):
+    """Test a capsule without registration module
+
+    :id: a2f23e42-648d-4428-a961-6e0b933c6dff
+
+    :steps:
+
+            1. Get a configured capsule
+            2. Set the registration to false for the capsule
+            3. Verify the result
+
+    :expectedresults: Registration module should be set to false on capsule.
+    """
+    module_target_sat.api.SmartProxy().search(
+        query={'search': f'name = {module_capsule_configured.hostname}'}
+    )[0]
+    res = module_capsule_configured.install(
+        cmd_args={},
+        cmd_kwargs={'foreman-proxy-registration': 'false', 'foreman-proxy-templates': 'true'},
+    )
```

Comment:
by default, both of this will be disabled on capsule, so should we try registration without this step?

Reply:
Yes. removed it

Reply:
The capsule setup includes setting these variables so adding the command back.

Reply:
capsule setup includes setting these variables, I didn't get?

Reply:
Since I am using module scoped fixtures so explicitly setting these values.